specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json

```diff
@@ -2688,6 +2688,127 @@
+          "type": "string"
+        },
+        "promotionCode": {
+          "description": "The promotion code.",
+          "type": "string"
+        },
+        "version": {
+          "description": "Version of product.",
+          "type": "string"
+        }
+      }
+    },
+    "RemotePrivateEndpointConnection": {
+      "description": "A remote private endpoint connection",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "RemotePrivateEndpointConnection resource specific properties",
+          "properties": {
+            "provisioningState": {
+              "type": "string",
+              "readOnly": true
+            },
+            "privateEndpoint": {
+              "$ref": "./CommonDefinitions.json#/definitions/ArmIdWrapper",
+              "description": "PrivateEndpoint of a remote private endpoint connection"
+            },
+            "privateLinkServiceConnectionState": {
+              "$ref": "./CommonDefinitions.json#/definitions/PrivateLinkConnectionState"
+            },
+            "ipAddresses": {
+              "description": "Private IPAddresses mapped to the remote private endpoint",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ResponseMessageEnvelope[RemotePrivateEndpointConnection]": {
+      "description": "Message envelope that contains the common Azure resource manager properties and the resource provider specific content.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource Id. Typically ID is populated only for responses to GET requests. Caller is responsible for passing in this\nvalue for GET requests only.\nFor example: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupId}/providers/Microsoft.Web/sites/{sitename}",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of resource e.g \"Microsoft.Web/sites\".",
+          "type": "string"
+        },
+        "location": {
+          "description": "Geographical region resource belongs to e.g. SouthCentralUS, SouthEastAsia.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Tags associated with resource.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "plan": {
+          "$ref": "#/definitions/ArmPlan",
+          "description": "Azure resource manager plan."
+        },
+        "properties": {
+          "$ref": "#/definitions/RemotePrivateEndpointConnection",
```

Comment:
>RemotePrivateEndpointConnection [](start = 33, length = 31)

this contains an allOf with ProxyResource so there is id,name,type within the properties as well?